### PR TITLE
add Seqouia note to macOS download page

### PR DESCRIPTION
**Pull request by splatterxl**
With macOS Seqouia's Developer Build, the option to right click and force opening of a quarantined app has been removed. You now need to go to the Privacy & Settings section of System Settings and force it there.

ref: https://github.com/Vencord/Installer/issues/130#issuecomment-2176271476

image

### CODE REVIEW

**Comment by Vendicated** [comment]
thanks! it would be helpful to add a screenshot / quick video that shows how to do it

**Comment by splatterxl** [comment]
I added a screenshot to a `<details>` element. I also added a link to the Apple support page for Gatekeeper: https://support.apple.com/102445

File: src/components/pages/download/MacTab.astro [hunk 1]
```diff
@@ -13,9 +13,39 @@ import Tab from "./Tab.astro";
...
+                </a>
+            </p>
+
+            <a href="/assets/seqouia-install.png" target="_blank">
+                <img
+                    class="sequoia-install-img"
+                    width="700"
+                    height="auto"
+                    src="/assets/seqouia-install.png"
+                    alt="An image showing how to allow VencordInstaller.app to Open Anyway in Privacy & Security settings, on macOS Seqouia."
```
Comment (Vendicated) [inline src/components/pages/download/MacTab.astro:45]: you [should avoid](https://accessibility.huit.harvard.edu/describe-content-images) words like "an image" in alt texts, it's redundant

File: src/components/pages/download/MacTab.astro [hunk 2]
```diff
@@ -25,3 +55,21 @@ import Tab from "./Tab.astro";
...
+    .sequoia-install-img {
+        margin-top: 1rem;
+        border-radius: 0.5rem;
+    }
+
+    @media (max-width: 1300px) {
+        .sequoia-install-img {
+            width: 50vw;
+        }
+    }
```
Comment (Vendicated) [inline src/components/pages/download/MacTab.astro:74]: do we really need this? usually `img { max-width: 100%; }` does the trick

Comment (splatterxl) [reply src/components/pages/download/MacTab.astro:74]: accidentally left this in while i was testing different styles

**Comment by Vendicated** [comment]
thanks a lot :3

**Pull request merged**